x/wasm/keeper/submsg_test.go

```diff
@@ -294,15 +295,15 @@ func TestDispatchSubMsgErrorHandling(t *testing.T) {
 			msg:      validBankSend,
 			gasLimit: &subGasLimit,
 			// uses same gas as call without limit
-			resultAssertions: []assertion{assertReturnedEvents(5), assertGasUsed(134000, 137000)},
+			resultAssertions: []assertion{assertReturnedEvents(5), assertGasUsed(138000, 141000)},
```

Comment:
I'm going to go ahead and assume that this is here and changed because SDK 45 consumes gas differently from 44. If that's the case and you're sure these figures are correct, then everything is fine here